tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.alias.json",
   "compilerOptions": {
-    "lib": ["ESNext"],
+    "lib": ["ESNext", "dom"],
```

Comment:
After more reading, the lib attribute is, as the name implies, for specifying the libraries that will be used, specifically JS-related definitions provided by TS. The choice ESNext and dom were arbitrarily taken from the internet. I tested things out and we should go for ES6 which is the pretty much core of modern JavaScript (doesn't matter because of #64 )